sample/src/demo_multi.ts

```diff
@@ -33,7 +32,6 @@ const documents: any = {
   "did:example:489398593#test": keyPairOptions,
   "did:example:489398593": exampleControllerDoc,
   "https://w3id.org/security/bbs/v1": bbsContext,
-  "https://w3id.org/security/v3-unstable": securityV3,
```

Comment:
Does removing this mean we no longer support v3-unstable context in our library and the bbs/v1 has to be used? I'm good with that, but let's make sure secureKey goes in the same direction for interop reasons.

Maybe we add this context as a normative statement to the LDP-BBS spec?

Reply:
Na doesn't mean we dont support it just means its not required by the library to interact with the generated proofs

Reply:
We should discuss normative considerations around `@context` values in the [ldp-bbs](https://github.com/w3c-ccg/ldp-bbs2020) spec